apps/web/src/screens/Explore/LayoutCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { Image, Stack, Text } from '@walless/gui';
+
+export interface LayoutCardProps {
+	id: string;
+	name: string;
+	description: string;
+	thumbnail: string;
+	logo: string;
+	loveCount: number;
+	activeUsers: number;
+}
+
+const LayoutCard: FC<LayoutCardProps> = (props) => {
+	return (
+		<Stack
+			backgroundColor="#131C24"
+			width="100%"
+			height={259}
+			borderRadius={12}
+		>
+			<Image
+				src={props.thumbnail}
+				width="100%"
+				height={133}
+				borderTopLeftRadius={12}
+				borderTopRightRadius={12}
+			/>
+
+			<Stack
+				marginHorizontal={12}
+				marginVertical={16}
+				display="flex"
+				y={-36}
+				zIndex={1}
```

Comment:
is it necessary to use zIndex here?

Reply:
zIndex must be used here, otherwise the thumbnail will overlap the logo

Reply:
okay let it be for now, will check again with @fluctlight-kayaba